PurchasesTests/Attribution/AttributionTypeFactoryTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Created by Andrés Boedo on 2/25/21.
+// Copyright (c) 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import Nimble
+import Purchases
+
+class AttributionTypeFactoryTests: XCTestCase {
```

Comment:
The stuff here used to live in AttributionFetcherTests, but since I moved the `rot13` stuff to `AttributionTypeFactory`, I also moved the corresponding tests